### PR TITLE
Add missing params.opengraph locale for fr_FR, nl_NL

### PR DESCRIPTION
**Pull request by d-oit**
Add missing "params.opengraph" > "locale" for fr_FR, nl_NL instead of the default en_US: <meta property="og:locale" content="en_US" /

### CODE REVIEW

File: exampleSite/config/_default/languages.toml [hunk 1]
```diff
@@ -39,6 +41,8 @@ reference = "Meer projecten"
 languageName = "Français"
 contentDir = "content/fr"
 weight = 2
+[nl.params.opengraph]
```
Comment (markdumay) [inline exampleSite/config/_default/languages.toml:44]: Thanks for this PR @d-oit. However, I believe this line should be `[fr.params.opengraph]`?

Comment (markdumay) [reply exampleSite/config/_default/languages.toml:44]: Currently, the Netlify build fails, likely related to the above comment:

```
11:02:23 AM: Error: failed to load config: failed to unmarshal config for path "/opt/build/repo/exampleSite/config/_default/languages.toml": "_stream.toml:1:1": unmarshal failed: toml: table opengraph already exists
```

**Comment by d-oit** [comment]
Yes, I had problems if I switch between my notebook with git. Have to fix
it immediately.

Found also another config behavior - my lasted GitHub discussion.


Mark Dumay ***@***.***> schrieb am Mo., 22. Juli 2024, 11:11:

**Comment by d-oit** [comment]
Should I create a new patch for the whole config that I found out?

Dominik Oswald ***@***.***> schrieb am Mo., 22. Juli 2024, 11:50:

**Comment by markdumay** [comment]
> Should I create a new patch for the whole config that I found out?

If it's related to open graph, you could push it to this PR. Otherwise, I'd suggest to fix this patch and open a new PR for the other issue.

**Pull request merged**